src/Components/Page.js

```diff
@@ -0,0 +1,47 @@
+import React, { Fragment } from 'react'
```

Comment:
Initial version of Page component which can be reused for the other pages to save repetition of the loading spinner, error message, pages titles and breadcrumbs etc.